backend/command/test/items.e2e-spec.ts

```diff
@@ -224,6 +236,223 @@ describe('ItemsController (e2e)', () => {
+      // Create eventstore stream
+      mockEventStore.existingStreams = [
+        `${ALLOWED_EVENT_ENTITIES.ITEM}-${item.slug}`,
+      ];
+      //ACT
+      const res = await request(server)
+        .post(commandsPath + `items/${item.slug}/suggest/edit`)
+        .send({ image: 'test' });
+      // ASSERT
+      expect(res.status).toEqual(HttpStatus.OK);
+      // Does suggestion exist in mongoDb
+      const suggestion = await editSuggestionModel.findOne({
+        itemSlug: item.slug,
+      });
+      expect(suggestion.userId).toEqual(verifiedRequestUser.id);
+    });
+
+    // WARNING: The following tests are for testing the edit functionality itself. As of now this is triggered via the
+    // suggestion request, however this is subject to change. These tests are more relevant for grading than functionality
+
+    it('items/:slug/suggest/edit => Should update item', async () => {
+      // ARRANGE
+      const item = new itemModel(singleItem);
+      await item.save();
+      // Create eventstore stream
+      mockEventStore.existingStreams = [
+        `${ALLOWED_EVENT_ENTITIES.ITEM}-${item.slug}`,
+      ];
+      //ACT
+      const res = await request(server)
+        .post(commandsPath + `items/${item.slug}/suggest/edit`)
+        .send({ image: 'test' });
+      // ASSERT
+      expect(res.status).toEqual(HttpStatus.OK);
+      // This will pass if met or throw an error if callback condition is never met
+      await retryCallback(
+        async () =>
+          // Has item been updated?
+          (await itemModel.findOne({ slug: item.slug })).image === 'test',
+      );
+    });
+
+    it('items/:slug/suggest/edit => Should be able to set property to null', async () => {
+      // ARRANGE
+      const item = new itemModel(singleItem);
+      await item.save();
+      // Create eventstore stream
+      mockEventStore.existingStreams = [
+        `${ALLOWED_EVENT_ENTITIES.ITEM}-${item.slug}`,
+      ];
+      //ACT
+      const res = await request(server)
+        .post(commandsPath + `items/${item.slug}/suggest/edit`)
+        .send({ source: null });
+      // ASSERT
+      expect(res.status).toEqual(HttpStatus.OK);
+      // This will pass if met or throw an error if callback condition is never met
+      await retryCallback(
+        async () =>
+          // Has item been updated?
+          (await itemModel.findOne({ slug: item.slug })).source === null,
+      );
+    });
+
+    it('items/:slug/suggest/edit => Should be able to update weight (nested Object)', async () => {
+      // ARRANGE
+      const item = new itemModel({
+        ...singleItem,
+        weight: { value: 1123675e30, isCa: true },
+      });
+      await item.save();
+      // Create eventstore stream
+      mockEventStore.existingStreams = [
+        `${ALLOWED_EVENT_ENTITIES.ITEM}-${item.slug}`,
+      ];
+      //ACT
+      const updateWeigt = {
+        value: 2222222e30,
+        isCa: null,
+      };
+      const res = await request(server)
```

Comment:
This is a good test, but this test is missing:
existing weight: `weight: { value: 100, isCa: false, additionalWeight: 300 }`
suggestion:  `weight: { value: 200 }`
expected: `weight: { value: 200, isCa: false, additionalWeight: 300 }`

pretty sure this requires some logic change in the event